client/my-sites/checkout/checkout-thank-you/pending/index.tsx

```diff
@@ -144,7 +161,7 @@ function getSaaSProductRedirectUrl( receipt: ReceiptState ) {
 function useRedirectOnTransactionSuccess( {
 	orderId,
 	receiptId,
-	siteSlug,
+	siteSlug = '',
```

Comment:
What if we let this be `undefined` and then change `notifyAndPerformRedirect` so that it calls `sendMessageToOpener( siteSlug ?? '', ... )`? That way it's more clear that there might not be a siteSlug.